CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 1.4.5
+
+### Additions and Improvements
+
+ - Introduced Security Module Plugin API. This allows to switch to a different security module provider to 
+ provide cryptographic function that can be used by NodeKey (such as sign, ECDHKeyAgreement etc.). By default,
+  register KeyPairSecurityModule otherwise attempt to load Security Module via plugin API specified by 
```

Comment:
are we saying KeyPairSecurityModule is loaded by default?

Reply:
also would be nice to be consistent with casing and terms ie "Security Module" here vs "security module provider" above.